### ISSUE TITLE
Back navigation within activities screen after opening a single activity should'n trigger an activities reload

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/android#2536]
One issue I found testing the activities screen: 
When you click on an image to preview it (works just fine) and then hit the back button to get back to the activities list, it is basically reset and like on initial launch shows and empty list + loading indicator and starts loading the activities again.
I would have expected the app to simply display the existing already loaded activity items again and also have the scroller at that position while I can't say if the original implementation did this.

**Comment by tobiasKaminsky** [comment]
As far as I remember, we never had this included, but it is a nice improvement :+1:

**Comment by AndyScherzinger** [comment]
Yeah, could be called an enhancement or a bug, either way we should fix it :)
